### PR TITLE
Invitation link moved to confirmed registration screen

### PR DESCRIPTION
**Pull request by lucasferraro**
#4311  The idea is to not display the invitation link until the registration process has been completed. Now the Business Account registration flow has 2 steps instead of 3 and in the 'confirmed registration' screen now we are showing the invitation link section.

https://github.com/coopcycle/coopcycle-web/assets/4819244/73345850-17e8-4927-a72e-c69a9bb5614f

### CODE REVIEW

**Comment by Atala** [comment]
hello @lucasferraro i just made a comment, which is more a question

i have an UI glitch on the confirmation page can you fix it ? I think we should just show the "invitation link" part when we display the invitation link.

+ add a link below the invitation link to the "business account admin" is not stucked on this page

thx;

image

once it is done you can merge !

**Comment by lucasferraro** [comment]
> hello @lucasferraro i just made a comment, which is more a question
> 
> i have an UI glitch on the confirmation page can you fix it ? I think we should just show the "invitation link" part when we display the invitation link.
> 
>     * add a link below the invitation link to the "business account admin" is not stucked on this page
> 
> 
> thx;
> 
> image
> 
> once it is done you can merge !

@Atala 
What do you think about this approach? Instead of show a new link, for business account managers the "confirmation" button in the congrats message section will redirect to the business account settings page.


https://github.com/coopcycle/coopcycle-web/assets/4819244/47db6c50-4ecb-46d7-8d48-9eb662d2bd4d

**Comment by Atala** [comment]
Ok let s go for this

File: src/Form/BusinessAccountRegistrationForm.php [hunk 1]
```diff
@@ -51,16 +51,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
+
                 $code = $this->tokenGenerator->generateToken();
                 $invitationLink = $this->urlGenerator->generate('invitation_define_password', [
                     'code' => $code
                 ], UrlGeneratorInterface::ABSOLUTE_URL);
 
                 $builder
-                    ->add('invitationLink', UrlType::class, [
-                        'label' => 'registration.step.invitation.copy.link',
+                    ->add('invitationLink', HiddenType::class, [
```
[the remark this answers is not in the document]
Comment (lucasferraro) [reply src/Form/BusinessAccountRegistrationForm.php]: It was not needed indeed :) but removing it needed some changes. Already commited

**Pull request merged**